### ISSUE TITLE
notification should pop up when playing music

### ISSUE THREAD

**Issue by AbhinavParshad** [abh80/smart-edge#30]
When  playing music and you get a notification the audio should move to another small bubble and stay like that until the notifications are cleared

**Comment by mukeshwebs** [comment]
yes i also notices this problem when playing music notifications don't pop up

**Comment by abh80** [comment]
Its on priority i will add a option to manage priority of plugins
